ftdetect/fish.vim

```diff
@@ -7,7 +8,7 @@ autocmd BufRead *
             \ endif
 
 " Move cursor to first empty line when using funced.
-autocmd BufRead fish_funced_*_*.fish call search('^$')
+autocmd BufRead fish_funced* call search('^$')
```

Comment:
Did something in fish change to make this change in pattern necessary? And if it still names funced files `.fish` the above `setfiletype` is also superfluous.

Reply:
IIRC there was some version of `fish` that had a change/bug in naming style of the intermediate files created by `funced`. As it seems they have change it again or fixed the bug so this is probably both no longer necessary.

As it was working I have not reviewed if these changes are necessary anymore ever since. ;-)

Reply:
As you can see in this commig https://github.com/fish-shell/fish-shell/commit/5accc7c6c5b2234998d1a88de2dfeb1057bd5507#diff-64bebefe19224c5455df182dc4b55ac9 they changed the pattern. Also before it already was
`fish_funced.XXXXXXXX.fish`
where it is using a dot instead of underscore as a separation between fish_funced and the random part. So that's where the change in pattern came from.

By now they removed the `fish_funced*` prefix altogether.

Reply:
Hm, so now funced isn't part of the file name at all? Then I don't know how we can detect funced like we need.

Reply:
I cannot see where this is coming from but in all my tries with `funced` now I've got a first line like

```# Defined in /tmp/fish.XXXXXX/funcname.fish @ line 2```

I get this for existing functions as well as new functions. If this is a consistent pattern this could be used but it needs confirmation if this happens for other people as well.

I am using Arch Linux, `neovim` 0.2.2 and `fish` 2.6.0.